x-pack/metricbeat/module/mssql/performance/performance.go

```diff
@@ -65,6 +65,9 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 func (m *MetricSet) Fetch(reporter mb.ReporterV2) {
 	var err error
 	var rows *sql.Rows
+	var BufferCacheHitRatio = "Buffer cache hit ratio"
```

Comment:
```suggestion
	var BufferCacheHitRatio = "Buffer cache hit ratio"
```

Are you sure of the field name?